tidy3d/components/data/monitor_data.py

```diff
@@ -80,8 +80,9 @@ class AbstractFieldData(MonitorData, AbstractFieldDataset, ABC):
     grid_expanded: Grid = pd.Field(
         None,
         title="Expanded Grid",
-        description=":class:`.Grid` on which the symmetry will be expanded. "
-        "Required only if any of the ``symmetry`` field are non-zero.",
+        description=":class:`.Grid` discretization of the associated monitor in the simulation "
+        "which created the data. Required if symmetries are present, as "
```

Comment:
So I was thinking about doing that, but in the end decided against it mostly because for some quick testing it may be convenient not to have to create a grid. Basically there are a few places in our tests, as well as in our docstring examples, where we create a data structure without passing a `grid_expanded`. But maybe this is not a good enough reason and it would be safer to have it required?

Reply:
up to you. maybe we can warn if not supplied?

Reply:
Ok added a warning (also needed to update docstring tests as they erorr if a warning is produced as they expect no output. Not sure if possible to turn off, probably yes, but better have good examples in the docstrings I guess.)